### ISSUE TITLE
Quiz on Hyperparameter tuning, Batch Normalization, Programming Frameworks

### ISSUE THREAD

**Issue by jinzishuai** [jinzishuai/learn2deeplearn#24]
![image](https://user-images.githubusercontent.com/1074685/31871334-a8607e76-b770-11e7-9208-6ab51a69e41c.png)
![image](https://user-images.githubusercontent.com/1074685/31871369-e8f59660-b770-11e7-9e84-f9127340d4a7.png)

**Comment by jinzishuai** [comment]
gamma and beta are both vectors, just has the mu and sigma for the parameter initialize phrase.
